cmd/operator-sdk/add/crd.go

```diff
@@ -98,6 +96,14 @@ func crdFunc(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("crd scaffold failed: (%v)", err)
 	}
 
+	// This command does not consider an APIs dir. Instead it adds a plain CRD
+	// for the provided resource. We can use NewCRDNonGo to get this behavior.
+	gcfg := genutil.Config{InputDir: scaffold.ApisDir, OutputDir: scaffold.CRDsDir}
```

Comment:
For non-go, should `InputDir` be `scaffold.CRDsDir`? Also, since `NewCRDNonGo()` defaults the config's input and output dirs, can we pass an empty `genutil.Config{}`?

Ditto for other places that we're passing the config that use the defaults.

Reply:
Yes we can pass an empty config here, changing.